xsl/pretext-common.xsl

```diff
@@ -1512,7 +1512,9 @@ Book (with parts), "section" at level 3
     <xsl:variable name="latex-left-justified">
         <xsl:call-template name="sanitize-text">
             <xsl:with-param name="text">
-                <xsl:value-of select="$docinfo/macros" />
+                <xsl:for-each select="$docinfo/macros">
+                    <xsl:value-of select="." />
+                </xsl:for-each>
```

Comment:
An alternative to this XSL (since we try to rarely use `for-each`):

```
    <xsl:apply-templates select="$docinfo/macros">
```

That should be enough (but test it if you are inclined). The default template for the "macros" element will just apply templates to its child nodes, which are all text. And the the special (non-deefault) template we have for `text()` should end up just doing `<xsl:value-of select="." />` since there are no math elements inside the "macros" element. So, same thing in the end.